src/utils/ipfs.ts

```diff
@@ -9,7 +12,7 @@ export const postToIPFS = async (data: any): Promise<string> => {
       'Basic ' +
       btoa(process.env.NEXT_PUBLIC_INFURA_ID + ':' + process.env.NEXT_PUBLIC_INFURA_SECRET);
     ipfs = create({
-      url: 'https://infura-ipfs.io:5001/api/v0',
+      url: process.env.NEXT_PUBLIC_IPFS_BASE_URL,
```

Comment:
this will prevent IPFS to work, the public URL to read is not the same as the one to write

Reply:
I see
But when I was running it locally, I was able to write/read successfully